mesonbuild/mparser.py

```diff
@@ -90,6 +91,7 @@ def __init__(self, code):
         self.code = code
         self.keywords = {'true', 'false', 'if', 'else', 'elif',
                          'endif', 'and', 'or', 'not', 'foreach', 'endforeach'}
+        self.future_keywords = {'continue', 'break', 'in'}
```

Comment:
Should probably contain `return` too. Anything else?

Reply:
Do you have plans for `return` or is it just in case? Could be nicer name than subdir_done() maybe.

I've checked the list of C++/python keywords and I don't see any other name that could be interesting to us, assuming `while` and `for` are ruled out by design, right?

Reply:
Updated the PR to include `return`.